### PR TITLE
Bump 1.20.0

### PR DESCRIPTION
**Pull request by rhatdan**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Review by giuseppe — APPROVED** [review]
LGTM

**Review by saschagrunert — APPROVED** [review]

File: contrib/rpm/buildah.spec
Comment (TomSweeneyRedHat) [inline contrib/rpm/buildah.spec:29]: I think this should be 1.20.0

Comment (rhatdan) [reply contrib/rpm/buildah.spec:29]: This is the 1.20.1-dev patch branch and what setup.sh makes.

Comment (TomSweeneyRedHat) [reply contrib/rpm/buildah.spec:29]: nope, 1.20.0 is wrong too, looking

Comment (TomSweeneyRedHat) [reply contrib/rpm/buildah.spec:29]: 1.20.1-dev
is what it should be

Comment (rhatdan) [reply contrib/rpm/buildah.spec:29]: I had that the last time, and the tests blew up.

Comment (TomSweeneyRedHat) [reply contrib/rpm/buildah.spec:29]: The tests may need to be adjusted too, it's probably looking at buildah.go still and needs to change to  define/types.go

File: contrib/rpm/buildah.spec
Comment (TomSweeneyRedHat) [inline contrib/rpm/buildah.spec:104]: We can clean this up next time, but should be a blank line after this one.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
/hold 
on tests

**Comment by TomSweeneyRedHat** [comment]
STORAGE_DRIVER:vfs test failed, not bread crumbs, restarted.

**Comment by edsantiago** [comment]
Latest flake was #2628 (the veth one). Restarted.

**Pull request merged**
